react_native_fundamentals/presentation.html

```diff
@@ -2,43 +2,1137 @@
+•Text:
+A React component for displaying text. Text supports nesting, styling, and touch handling. (Nested text example). The <Text> element is unique relative to layout: everything inside is no longer using the Flexbox layout but using text layout. This means that elements inside of a <Text> are no longer rectangles, but wrap when they see the end of the line.(Containers
+Example). Also, you need to know, that you must wrap all the text nodes inside of a <Text> component. You cannot have a text node directly under a <View>.
+•Image:
+A React component for displaying different types of images, including network images, static resources, temporary local images, and images from local disk, such as the camera roll. Image components has source prop. It is the image source (either a remote URL or a local file resource). This prop can also contain several remote URLs, specified together with their width and height and potentially with scale/other URI arguments. The native side will then choose the best uri to display based on the measured size of the image container
+
+•TextInput:
+A foundational component for inputting text into the app via a keyboard. Props provide configurability for several features, such as auto-correction, auto-capitalization, placeholder text, and different keyboard types, such as a numeric keypad. You can subscribe to the onChangeText events to read the user input. Other important props are the value and the placeholder. value: The value to show for the text input. TextInput is a controlled component, which means the native value will be forced to match this value prop if provided. Placeholder: 
+the string that will be rendered before text input has been entered. 
+•ScrollView:
+Component that wraps platform ScrollView while providing integration with touch locking "responder" system.
+Keep in mind that ScrollViews must have a bounded height in order to work, since they contain unbounded-height children into a bounded container (via a scroll interaction). In order to bound the height of a ScrollView, either set the height of the view directly (discouraged) or make sure all parent views have bounded height. 
+          </aside>
+        </section>
+
+        <section class="compare-slide">
+          <h4 class="slide-name">Using List Views</h4>
+          <div>
+          <div>
+            <img src="assets/images/flat-list.gif" alt="simple" />
+            <ul>
+              <li>long lists of data</li>
+              <li>renders items lazily</li>
+            </ul>
+          </div>
+          <div>
+            <img src="assets/images/scroll-view.gif" alt="simple" />
+            <ul>
+              <li>small number of things of a limited size</li>
+              <li>renders all its child components at once</li>
+            </ul>
+          </div>
+        </div>
+          <aside class="notes">
```

Comment:
Could you make some words with strong style?

Reply:
Done